src/components/notificationsCenter/notificationsCenter.styles.scss

```
@@ -361,23 +361,23 @@
 		font-size: 12px;
 		font-weight: 700;
-		color: #374151;
+		color: var(--m3-on-surface-variant, #374151);
 	}
 
 	&__chatPreviewText {
 		font-size: 13px;
-		color: #374151;
+		color: var(--m3-on-surface-variant, #374151);
 		line-height: 1.35;
 		word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)